app/controllers/chat_rooms_controller.rb

```diff
@@ -12,7 +12,7 @@ def create
         @chat_room = current_user.chat_rooms.build(chat_room_params)
         if @chat_room.save
             flash[:success] = 'Chat Room added!'
-            redirect_to welcome_index
+            redirect_to welcome_index_url
```

Comment:
generally you want to use the _path version unless you have a specific need for the _url version (usually within emails or things that are clicked on other sites). It doesn't hurt, just adds redundant info to the link.